test/extended/image_ecosystem/mysql_replica.go

```diff
@@ -13,7 +13,7 @@ import (
 	testutil "github.com/openshift/origin/test/util"
 
 	kapiv1 "k8s.io/api/core/v1"
-	//metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
?

Reply:
the code that was commented out (and is being renabled in this PR) uses this package, so the package import is being un-commented out.